### DISCUSSION TITLE
you can add item for count items in array

### DISCUSSION THREAD

**Discussion by alirezamhi** [mockapi-io/docs#1] [Ideas]
I want to add button pagination from my task but I can not do it because I do not have a number about all item in data 
I want you add this

**Comment by nsmirn0v** [comment]
Hi there! There is a way to customize API response to include item count. Take a look at this wiki and let me know if this works for you https://github.com/mockapi-io/docs/wiki/Custom-responses

**Reply by alirezamhi** [reply]
Thank you
